packages/frontity-org-theme/src/processors/borders.ts

```diff
@@ -0,0 +1,50 @@
+import { css } from "frontity";
+
+function createBordersProcessors(obj: {
+  [prop: string]: ReturnType<typeof css>;
+}) {
+  return Object.entries(obj).map(([className, style]) => ({
+    name: "borders",
+    test: element => {
+      return (
+        element.type === "element" &&
+        element.props.className?.split(/ +/).includes(className)
```

Comment:
I used the same RegExp for other processors, but I ended changing it by just `" "` (it would work the same at the end).